### PR TITLE
Test smoothn with fixed smoothing order

### PR DESCRIPTION
**Pull request by timspainUCL**
Another `smoothn()` test.

### CODE REVIEW

File: kaska/tests/test_smoothn.py [hunk 1]
```diff
@@ -15,29 +15,52 @@
...
+
+    assert np.abs(np.max(np.abs(res)) - target) < prec
     
-    unrobust_target = 0.7503929639274534
-    robust_target = 0.18943656067148762
     
-    #prec = 1e-12
+def test_fixed_order():
+    unrobust_target = 0.7503929639274534
+    unrobust_order = 56.93236088601813
```
Comment (mattgillucl) [inline kaska/tests/test_smoothn.py]: What are these magic numbers in `test_fixed_order()` ?

Comment (timspainUCL) [reply kaska/tests/test_smoothn.py]: The parameters and results for the test? I can add a comment to that effect.

**Comment by timspainUCL** [comment]
Everything from 2f08fab onwards (inclusive) is to try to diagnose and then correct the dodgy assertions.

**Comment by mattgillucl** [comment]
Is the dodgy assertions issue connected with the "precision tolerance varying between machines" issue? Is it the same issue in fact?

**Comment by dpshelio** [comment]
> E251 looks like a pointless moan. Having spaces around equals signs makes the code clearer. I wonder if we can switch off E251 on PEP8.

No! That's a strong convention within Python. Keyword arguments on a function don't use spaces to distinguish them from variable assignments. In R, however, they think otherwise, but they use the `<-` for the assignment operator instead.

**Comment by timspainUCL** [comment]
These tests will be included as part of the smoothn3 branch

**Comment by mattgillucl** [comment]
"Keyword arguments on a function don't use spaces to distinguish them from variable assignments."

But keyword arguments are enclosed within a pair of brackets, so it should be obvious which are keyword args and which are variable assignments.

**Comment by timspainUCL** [comment]
Maybe so, but we are working in python, so we should follow PEP8. And PEP8 suggests no spaces around keyword arguments.

**Comment by mattgillucl** [comment]
Personally I think the code is clearer with the spaces, but I guess that's not going to happen.

**Pull request closed without merging**